pkgs/applications/terminal-emulators/darktile/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, buildGoModule
+, fetchFromGitHub
+, lib
+
+, go
+, pkg-config
+
+, libX11
+, libXcursor
+, libXrandr
+, libXinerama
+, libXi
+, libXext
+, libXxf86vm
+, libGL }:
+
+let
+  deps = [
+    go
+    pkg-config
+    libX11
+    libXcursor
+    libXrandr
+    libXinerama
+    libXi
+    libXext
+    libXxf86vm
+    libGL
+  ];
+in
+stdenv.mkDerivation rec {
+  pname = "darktile";
+  version = "0.0.10";
+
+  src = fetchFromGitHub {
+    owner = "liamg";
+    repo = "darktile";
+    rev = "v${version}";
+    sha256 = "0pdj4yv3qrq56gb67p85ara3g8qrzw5ha787bl2ls4vcx85q7303";
+  };
+
+  buildInputs = deps;
+
+  preBuild = ''
+    substituteInPlace scripts/build.sh \
+      --replace "bash" "sh"
+    makeFlagsArray+=(HOME=$TMP)
+  '';
+
+  # Darktile does not have an 'install' Makefile target,
+  # so the installation is done in postBuild.
+  postBuild = ''
+    mkdir -p $out/bin
+    mv darktile $out/bin
+  '';
+  dontInstall = true;
+
+  meta = with lib; {
+    description = "A GPU rendered terminal emulator designed for tiling window managers";
+    homepage = "https://github.com/liamg/darktile";
+    downloadPage = "https://github.com/liamg/darktile/releases";
+    changelog = "https://github.com/liamg/darktile/releases/tag/v${version}";
+    license = license.mit;
+    maintainers = with maintainers; [ flexagoon ];
```

Comment:
You should add yourself to [`maintainers/maintainer-list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) with a separate commit `maintainers: add flexagoon`.

Reply:
I already did that in my other pull request
https://github.com/NixOS/nixpkgs/pull/135948